src/main/java/com/google/research/bleth/simulator/IObserver.java

```diff
@@ -0,0 +1,19 @@
+package com.google.research.bleth.simulator;
+
+/** Represent an observer. */
+public interface IObserver extends Agent {
+
+    /**
+     * Add a transmission to the transmissions the observer observed so far in the current round.
+     * @param beaconTransmission the new transmission the observer observe.
+     */
+    void observe(Transmission beaconTransmission);
+
+    /**
+     * Deliver all the transmissions the observer observed in the current round to its resolver.
+     */
+    void passInformationToResolver();
+
+    /** Returns true if the observer is active, false otherwise. */
+    boolean isObserverAwake();
```

Comment:
I my opinion
```java
observer.isAwake();
```
Would be better than
```java
observer.isObserverAwake();
```

Because it is already clear that this methods refers to an observer (but up to you).